modules/admission-webhook-create.adoc

```diff
@@ -0,0 +1,376 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-create_{context}"]
+= Creating an admission webhook
+
+This procedure outlines high level steps to deploy a webhook server as a DaemonSet and to deploy the webhook server as an aggregated API server. Deploying the webhook server as an aggregated API server allows other {product-title} components to communicate with the webhook using internal credentials and facilitates testing using the `oc` command. This also enables role based access control (RBAC) into the webhook and prevents token information from other API servers being disclosed to the webhook. The API server calls `kube-apiserver` with its fully authenticated service account. The aggregated API server then calls to the webhook using the request header certificate. The webhook verifies incoming client certificates through the published request header aggregation CA.
+
+[NOTE]
+====
+The link:https://github.com/openshift/kubernetes-namespace-reservation[kubernetes-namespace-reservation] repository provides a full end-to-end example of a webhook server and aggregated API server deployment. Note that this repository has not been fully tested or validated by Red Hat.
+====
+
+.Procedure
+
+. Build a webhook server container image and make it available to the cluster using an image registry.
+
+. Create a local CA key and certificate and use them to sign the webhook server's certificate signing request (CSR).
+
+. Create a new namespace for webhook resources:
++
+----
+$ oc create namespace my-webhook-namespace
+----
+
+. Switch to the new namespace:
```

Comment:
@bergerhoffer https://www.reddit.com/r/openshift/comments/78jcpk/namespaces_vs_projects/

Reply:
In all honesty, I did not realise there is a difference between namespaces and projects until I read this. I have changed the step to create a project instead of a namespace, as suggested.

I have also added an appropriate prerequisite, because cluster administration privileges are required.